src/main/kotlin/org/rust/lang/core/types/ty/TyTraitObject.kt

```diff
@@ -54,6 +55,9 @@ class TyTraitObject(
     fun getTraitBoundsTransitively(): Collection<BoundElement<RsTraitItem>> =
         traits.flatMap { it.flattenHierarchy }
 
+    val nonAutoTrait: RsTraitItem?
```

Comment:
```suggestion
    val baseTrait: RsTraitItem?
```
Accoring to [The Rust Reference](https://doc.rust-lang.org/reference/types/trait-object.html)